### PR TITLE
Dashboard add bswh dev sso config

### PR DESCRIPTION
**Pull request by Gbarra9**
This Pull request is related to 

- https://github.com/episphere/connect/issues/884

Code Changes:
Have SSO pop-up window show for BSWH in dev

Screenshot 2024-05-07 at 11 04 12 AM

### CODE REVIEW

**Review by anthonypetersen — APPROVED** [review]

File: siteManagerDashboard/dev/identityProvider.js [hunk 1]
```diff
@@ -1,37 +1,41 @@
 export const SSOConfig = (inputValue) => {
     let tenantID = '';
     let provider = '';
-    if(/nih.gov/i.test(inputValue)) {
+    if (/nih.gov/i.test(inputValue)) {
         tenantID = 'NIH-SSO-qfszp';
         provider = 'saml.nih-sso';
     };
```
Comment (we-ai) [inline siteManagerDashboard/dev/identityProvider.js:7]: No need `;` outside curly bracket, after if condition check. Same for other cases below.
```suggestion
    }
```

File: siteManagerDashboard/dev/identityProvider.js [hunk 2]
```diff
@@ -1,37 +1,41 @@
...
+    if (/marshfieldresearch.org/i.test(inputValue) || /marshfieldclinic.org/i.test(inputValue)) {
         tenantID = 'MFC-SSO-fljvd';
         provider = 'saml.connect-mfc'
     }
-    return {tenantID, provider}
+    if (/bswhealth.org/i.test(inputValue)) {
+        tenantID = 'BSWH-SSO-y2jj3';
+        provider = 'saml.connect-bswh';
+    };
+    return { tenantID, provider }
```
Comment (we-ai) [inline siteManagerDashboard/dev/identityProvider.js:40]:
```suggestion

    return { tenantID, provider };
```

**Comment by we-ai** [comment]
The condition checks will run from start to end, even when there's a match found. Although there's very small impact for short code like this, it can be better to immediate return result when a match is found.

**Review by we-ai — APPROVED** [review]
Commented on minor things.

**Pull request merged**